src/libraries/System.Security.Principal.Windows/src/System.Security.Principal.Windows.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;netstandard2.0;netcoreapp2.0-windows;netcoreapp2.0-Unix;netcoreapp2.1-windows;netcoreapp2.1-Unix;net461-windows</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;netstandard2.0;netstandard2.0-windows;netstandard2.0-Unix;netcoreapp2.1-windows;netcoreapp2.1-Unix;net461-windows</TargetFrameworks>
```

Comment:
Why are we changing the tfm?

Reply:
The reason behind is to have netstandard2.0 implementation for uap.

Reply:
but why are we doing this? uap isn't a configuration we are actively working on anymore.

Reply:
we are still running package testing for it.

Reply:
IMO we should remove package testing in master for it and/or remove the uap assets in our packages. (tomorrow's meeting)

Reply:
cc @ericstj